doc/sphinxman/source/sapt.rst

```diff
@@ -10,8 +10,8 @@
 SAPT: Symmetry-Adapted Perturbation Theory
 ==========================================
 
-.. codeauthor:: Edward G. Hohenstein and Rob M. Parrish
-.. sectionauthor:: Edward G. Hohenstein
+.. codeauthor:: Edward G. Hohenstein, Rob M. Parrish and Jerome F. Gonthier
+.. sectionauthor:: Edward G. Hohenstein and Jerome F. Gonthier
```

Comment:
You're welcome to accent yourself (https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/abbr_accents.rst) in the docs, if you like. `startofword\ |aa|\ restofword` for intra-word accents.